### PR TITLE
i18n(ko-KR): create `tabs.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description

- create `tabs.mdx`

### CODE REVIEW

**Comment by jsparkdev** [comment]
When https://github.com/withastro/starlight/pull/2363 is successfully merged, I expect any errors that occur in this PR to be resolved.

File: docs/src/content/docs/ko/components/tabs.mdx [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+[Starlight의 내장 아이콘 중 하나](/ko/reference/icons/#모든-아이콘)의 이름으로 설정된 [`icon`](#icon) 속성을 사용하여 탭 항목에 아이콘을 포함하면 레이블 옆에 아이콘이 표시됩니다.
+
+<Preview>
+
+```mdx /icon="\w+"/
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<Tabs>
+	<TabItem label="별" icon="별">
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/components/tabs.mdx:156]:
```suggestion
	<TabItem label="별" icon="star">
```

A bit too much translated 😄

File: docs/src/content/docs/ko/components/tabs.mdx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+		시리우스, 베가, 베텔게우스
+	</TabItem>
+	<TabItem label="달" icon="달">
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/components/tabs.mdx:159]:
```suggestion
	<TabItem label="달" icon="moon">
```

File: docs/src/content/docs/ko/components/tabs.mdx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+		이오, 유로파, 가니메데
+	</TabItem>
+</Tabs>
+```
+
+<Fragment slot="markdoc">
+
+```markdoc /icon="\w+"/
+{% tabs %}
+{% tabitem label="별" icon="별" %}
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/components/tabs.mdx:169]:
```suggestion
{% tabitem label="별" icon="star" %}
```

File: docs/src/content/docs/ko/components/tabs.mdx [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+시리우스, 베가, 베텔게우스
+{% /tabitem %}
+
+{% tabitem label="달" icon="달" %}
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/components/tabs.mdx:173]:
```suggestion
{% tabitem label="달" icon="moon" %}
```

File: docs/src/content/docs/ko/components/tabs.mdx [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+이오, 유로파, 가니메데
+{% /tabitem %}
+{% /tabs %}
+```
+
+</Fragment>
+
+<Tabs slot="preview">
+	<TabItem label="별" icon="별">
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/components/tabs.mdx:182]:
```suggestion
	<TabItem label="별" icon="star">
```

File: docs/src/content/docs/ko/components/tabs.mdx [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+		시리우스, 베가, 베텔게우스
+	</TabItem>
+	<TabItem label="달" icon="달">
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/components/tabs.mdx:185]:
```suggestion
	<TabItem label="달" icon="moon">
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). Left a few suggestions to fix icons but other than that, this looks good to me.

File: docs/src/content/docs/ko/components/tabs.mdx (hunk 1 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/components/tabs.mdx]: Thank you! I've fixed it.

**Review by HiDeoo — APPROVED** [review]
Amazing, thanks for fixing, looks ready to me :rocket:

**Pull request merged**